modules/acquia_cms_common/src/EventSubscriber/KernelTerminate/AcquiaConnectorTelemetryOverride.php

```diff
@@ -17,17 +21,143 @@
 class AcquiaConnectorTelemetryOverride extends AcquiaTelemetry {
 
   /**
-   * Sends Telemetry on a daily basis. This occurs after the response is sent.
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents() {
```

Comment:
@deepakmishra2 This isn't required as we already extended AcquiaTelemetry Class

cc: @vishalkhode1

Reply:
done